.github/ISSUE_TEMPLATE/report-a-bug.md

```diff
@@ -39,7 +39,7 @@ If you are reporting a bug, please follow the following steps:
 
 * **Server version:** <!-- Insert output of `/version` command -->
 
-* **Full output of `/mv version -b`:** <!-- Replace this with the command output's https://pastebin.com link -->
+* **Full output of `/mv version -p`:** <!-- Replace this with the command output's https://pastebin.com link -->
```

Comment:
Here too.

Reply:
ah crap i need to change for all 5 repo lol